lib/rubocop/cop/rspec/repeated_example_group_body.rb

```diff
@@ -34,6 +34,15 @@ module RSpec
       #      it { cool_predicate }
       #    end
       #
+      #    # good
+      #    context Array do
+      #      it { described_class.cool_predicate }
```

Comment:
Maybe `expect(described_class).to something`?
It's not a proper one-liner though, and I understand your intention to fit a `described_class` here to indicate what is the reason for those two identical group bodies to be considered different.
Can you think of a good example to fulfill both of those requirements?